extensions/data-plane/data-plane-google-storage/src/main/java/org/eclipse/edc/connector/dataplane/gcp/storage/validation/StringFunctions.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.dataplane.gcp.storage.validation;
+
+import org.jetbrains.annotations.Nullable;
+
+class StringFunctions {
+    public static boolean isEmpty(@Nullable String str) {
```

Comment:
in the edc util (that is set as dependency in this extension) we have the same method, called `StringUtils.isNullOrBlank`

Reply:
not quite the same though, as the method you mentioned does not `.trim()`.